nomad/state/schema.go

```diff
@@ -1230,10 +1230,71 @@ func secureVariablesTableSchema() *memdb.TableSchema {
 					},
 				},
 			},
+			"key_id": {
```

Comment:
Is there any benefit in making this an exported constant?

Reply:
Done in 2fa6a473f9b772c62f13f67ec473f4db197d330f